script/cluster-up-minikube.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# From minikube howto
+export MINIKUBE_WANTUPDATENOTIFICATION=false
+export MINIKUBE_WANTREPORTERRORPROMPT=false
+export MINIKUBE_HOME=$HOME
+export CHANGE_MINIKUBE_NONE_USER=true
+mkdir -p ~/.kube
+touch ~/.kube/config
+
+export KUBECONFIG=$HOME/.kube/config
+export PATH=${PATH}:${GOPATH:?}/bin
+
+MINIKUBE_VERSION=v0.21.0
+
+install_bin() {
+    local exe=${1:?}
+    test -n "${TRAVIS}" && sudo install -v ${exe} /usr/local/bin || install ${exe} ${GOPATH:?}/bin
+}
+
+# Travis ubuntu trusty env doesn't have nsenter, needed for VM-less minikube 
+# (--vm-driver=none, runs dockerized)
+check_or_build_nsenter() {
+    which nsenter >/dev/null && return 0
+    echo "INFO: Building 'nsenter' ..."
+cat <<-EOF | docker run -i --rm -v "$(pwd):/build" ubuntu:14.04 >& nsenter.build.log
+        apt-get update
+        apt-get install -qy git bison build-essential autopoint libtool automake autoconf gettext pkg-config
+        git clone --depth 1 git://git.kernel.org/pub/scm/utils/util-linux/util-linux.git /tmp/util-linux
+        cd /tmp/util-linux
+        ./autogen.sh
+        ./configure --without-python --disable-all-programs --enable-nsenter
+        make nsenter
+        cp -pfv nsenter /build
+EOF
+    if [ ! -f ./nsenter ]; then
+        echo "ERROR: nsenter build failed, log:"
+        cat nsenter.build.log
+        return 1
+    fi
+    echo "INFO: nsenter build OK, installing ..."
+    install_bin ./nsenter
+}
+check_or_install_minikube() {
+    which minikube || {
+        wget --no-clobber -O minikube \
+            https://storage.googleapis.com/minikube/releases/${MINIKUBE_VERSION}/minikube-linux-amd64
+        install_bin ./minikube
+    }
+}
+
+# Install nsenter if missing
+check_or_build_nsenter
+# Install minikube if missing
+check_or_install_minikube
+MINIKUBE_BIN=$(which minikube)
+
+# Start minikube
+sudo -E ${MINIKUBE_BIN} start --vm-driver=none \
+    --extra-config=apiserver.Authorization.Mode=RBAC
```

Comment:
Seems nopes - just tried running the test suite w/o above `--extra-config=...`,
failed with:
```
 ✗ Test simple function failure without RBAC rules
   (from function `k8s_wait_for_pod_logline' in file tests/../script/libtest.bash, line 51,
    from function `_wait_for_kubeless_controller_logline' in file tests/../script/libtest.bash, line 130,
    from function `test_must_fail_without_rbac_roles' in file tests/../script/libtest.bash, line 179,
    in test file tests/integration-tests.bats, line 18)
     `test_must_fail_without_rbac_roles' failed
 [...]
```